CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Consensus
+
+Since this config is used in code bases that affect various internal and external repos.
+It can potentially imply changes on repos with multi-million users so we need to be mindful
```

Comment:
I get the point. However I was thinking about the project I'm working on (SPiD SDK.JS) which has a couple of million users so we are very careful with every line of code. A simple change to the rule set may require us to touch many lines of code, which potentially can introduce security vulnerability to the whole schibsted (or the brands that use us like FINN, AB, Blocket, etc.)
But maybe I didn't write it in a good way? I just wanted to say: "don't cowboy around, these configs are gonna affect sensitive code". 💭 ?